ext/sodium/libsodium.c

```diff
@@ -1662,20 +1606,24 @@ PHP_FUNCTION(sodium_crypto_pwhash)
 		sodium_remove_param_values_from_backtrace(EG(exception));
 		RETURN_THROWS();
 	}
-	if (hash_len <= 0 || hash_len >= 0xffffffff) {
-		zend_throw_exception(sodium_exception_ce, "hash length must be greater than 0", 0);
+	if (hash_len <= 0) {
+		zend_argument_error(sodium_exception_ce, 1, "must be greater than 0");
+		RETURN_THROWS();
+	}
+	if (hash_len >= 0xffffffff) {
+		zend_argument_error(sodium_exception_ce, 1, "is too long");
```

Comment:
```suggestion
		zend_argument_error(sodium_exception_ce, 1, "is too large");
```

Unlike the similar message below, this is for an integer argument.